irods/manager/user_manager.py

```diff
@@ -310,7 +310,7 @@ def create(
         if MessageClass is UserAdminRequest:
             message_body = MessageClass("mkgroup", name, user_type, user_zone)
         else:
-            message_body = MessageClass("add", "user", name, user_type, "", "")
+            message_body = MessageClass("add", ("user" if self.sess.server_version < (5,) else "group"), name, user_type, "", "")
```

Comment:
For iRODS 4.3.3 and earlier, pass `user` and `rodsgroup`.
For iRODS 4.3.4 and later, pass `group` and an empty string for the user type.

Reply:
Does that apply for the UserAdmin API as well?

Reply:
ok. will modify approach slightly

Reply:
this is done.